domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/IllegalFieldValueCombination.java

```diff
@@ -33,6 +33,33 @@ public IllegalFieldValueCombination(final String filename, final String fieldNam
         putNoticeSpecific(KEY_CONFLICTING_FIELD_NAME, conflictingFieldName);
     }
 
+    public IllegalFieldValueCombination(final String filename, final String fieldName,
+                                        final String conflictingFieldName, final String entityId,
+                                        final String compositeKeyFirstPart, final String compositeKeySecondPart,
+                                        final String compositeKeyThirdPart, final String compositeKeyFourthPart,
+                                        final Object compositeKeyFirstValue, final Object compositeKeySecondValue,
+                                        final Object compositeKeyThirdValue, final Object compositeKeyFourthValue) {
+        super(filename, E_019,
+                "Conflicting field values",
+                "Conflicting field values for fields:`" + fieldName + "` and field:`" + conflictingFieldName
+                        + "` for entity with composite id:" + System.lineSeparator() +
```

Comment:
Not sure about the System.lineSeparator. Our JSON output should not contain formatting characters, so they should not be used here, or stripped before the  output. Though I'd be inclined to simply never put them in in the first place.

Reply:
As discussed, this will be removed.